server/database/db_build.sql

```diff
@@ -1,111 +1,73 @@
 BEGIN;
 
-DROP TABLE IF EXISTS users,
-roles,
-user-roles,
-jobs,
+CREATE OR REPLACE FUNCTION trigger_set_timestamp()
```

Comment:
This is a trigger, when triggered it will create a timestamp with the date now, useful for many parts , but im using it now only for groups and messages